src/TestSuite/Fixture/ChecksumTestFixture.php

```diff
@@ -118,6 +118,6 @@ protected function _hash(ConnectionInterface $connection): string
      */
     protected function _getTableKey(): string
     {
-        return $this->connection() . '-' . $this->table;
+        return $this->connection() . '-' . $this->table . '-' . static::class;
```

Comment:
It would be good to have a test for that.

Reply:
Currently, there are not tests.

Thus, it's acceptable for this PR to not add some.